ngraph/test/CMakeLists.txt

```diff
@@ -457,7 +457,7 @@ endif()
 target_link_libraries(unit-test PRIVATE ngraph_test_util)
 target_link_libraries(unit-test PRIVATE ngraph ngraph_backend libgtest)
 
-if (NGRAPH_ONNX_IMPORT_ENABLE)
+if (NGRAPH_ONNX_IMPORT_ENABLE AND NOT NGRAPH_USE_PROTOBUF_LITE)
```

Comment:
Does this mean that unit tests won't work with protobuf lite?

Reply:
Protobuf-lite does not support models in prototxt format.
Since most of these tests are in this format I decided to disable them.

Reply:
Got it. Can you leave a comment explaining it in this file?